addons/info/src/index.js

```diff
@@ -98,5 +99,8 @@ export const withInfo = makeDecorator({
 export { Story };
 
 export function setDefaults(newDefaults) {
-  return Object.assign(defaultOptions, newDefaults);
+  return deprecate(
```

Comment:
I don't feel like this is how I should've implemented the depreciation, but only this worked for me for some reason.